### PR TITLE
Fluid 5809 - merged changes from docs-template

### PR DESCRIPTION
**Pull request by jhung**
Merged in changes from docs-template which fixes the Topics skip link not hiding when the sidebar is collapsed.

@acheetham can you take a look at this after Pull #1 ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ This project contains the content needed to build and deploy a copy of the Inclu
 4. Run docpad: `docpad run`
 5. Confirm everything is working by loading `http://localhost:9778/` in a web browser.
 
-# Getting Updates for Your Custom Site
+# Getting Updates from docs-template for the Inclusive Learning Design Handbook
 
-Periodically `docs-template` will have updates. To update:
+Periodically `docs-template` will have updates. It is suggested you keep up to update with these changes:
```
Comment (acheetham) [inline README.md]: This should probably read "up to date" and not "up to update"

File: README.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ git merge docs-template/master
 npm update
 ```
 
+Conflicts are expected to occur when merging in changes from `docs-template` to your custom site. Manually resolve each conflict.
```
Comment (acheetham) [inline README.md]: Perhaps say "Conflicts may occur..." instead?

Comment (jhung) [inline README.md]: I've updated the Readme to be less pessimistic.

Comment (acheetham) [inline README.md]: :-)

**Pull request merged**